align_system/algorithms/llama_2_single_kdma_adm.py

```diff
@@ -13,29 +15,50 @@
 from align_system.utils import logging
 
 
+from align_system.similarity_measures import build_force_choice_func
+
+
 log = logging.getLogger(__name__)
 JSON_HIGHLIGHTER = JSONHighlighter()
 
-
+# TODO make this configurable from the config
 kdmas = {
     'basic_knowledge',
     'fairness',
     'protocol_focus',
+    'time_pressure',
     'risk_aversion',
     'utilitarianism',
     'mission',
     'denial',
+    'moral_deservingness',
+    'lives_saved',
+    'continuation_of_care'
 }
 
 kdma_remapping = {
     'basicknowledge': 'basic_knowledge',
     'protocolfocus': 'protocol_focus',
     'riskaversion': 'risk_aversion',
+    'moraldeservingness': 'moral_deservingness',
+    'continuationofcare': 'continuation_of_care',
+    'livesaved': 'lives_saved',
+    'timepressure': 'time_pressure',
 }
 
+# default_system_messages_path=os.path.join(
+#     pathlib.Path(__file__).parent.absolute(), '..',
+#     'prompt_engineering/bbn_alignment_system_messages_v1')
+
+# NOTE temporary way to change which system messages are used
+# TODO make this configurable from the config
 default_system_messages_path=os.path.join(
     pathlib.Path(__file__).parent.absolute(), '..',
-    'prompt_engineering/bbn_alignment_system_messages_v1')
+    'prompt_engineering/single_kdma_adm_system_messges')
```

Comment:
If all the system messages are in the same directory then being able to configure them is less important... configuration could still be useful if we're exploring using different system messages for the same KDMA